### PR TITLE
Added cleaned up code from tidying example.

### PR DESCRIPTION
**Pull request by ethanhsmith**

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            <p>The CSS Zen Garden invites you to relax and meditate on the important lessons of the masters. Begin to see with clarity. Learn to use the time-honored techniques in new and invigorating fashion. Become one with the web.</p>
+        </div>
+
+      </section>
+
+      <div class="main supporting" id="zen-supporting" role="main">
+        <div class="explanation" id="zen-explanation" role="article">
+          <h3>So What is This About?</h3>
+          <p>There is a continuing need to show the power of <abbr title="Cascading Style Sheets">CSS</abbr>. The Zen Garden aims to excite, inspire, and encourage participation. To begin, view some of the existing designs in the list. Clicking on any one will load the style sheet into this very page. The <abbr title="HyperText Markup Language">HTML</abbr> remains the same, the only thing that has changed is the external <abbr title="Cascading Style Sheets">CSS</abbr> file. Yes, really.</p>
```
Comment (Timothy-Tolley) [inline pristine-zen-garden.html]: I think your paragraph tags need to be on a separate line to the content with the content indented. For example:
<p>
     content
</p>

Comment (Timothy-Tolley) [reply pristine-zen-garden.html]: < p>
 --content 
</ p>

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+              </li>
+              </ul>
+            </nav>
+          </div>
+
+          <div class="zen-resources" id="zen-resources">
+            <h3 class="resources">Resources:</h3>
+            <ul>
+              <li class="view-css">
+              <a href="/214/214.css" title="View the source CSS file of the currently-viewed design.">
```
Comment (Timothy-Tolley) [inline pristine-zen-garden.html]: I think this content should be indented within the "li" tag :)

**Comment by ethanhsmith** [comment]
Thanks for your helpful comments Timothy. I've fixed all those lines up now 👍

**Review by Timothy-Tolley — APPROVED** [review]
Looks great mate! Will merge your branches after check in

**Comment by ethanhsmith** [comment]
Thank you 👍

**Pull request merged**